docs/user-guide/migrate-to-greptimedb/migrate-from-mysql.md

```diff
@@ -0,0 +1,98 @@
+# Migrate from MySQL
+
+This document will guide you through the migration process from MySQL to GreptimeDB.
+
+## Before you start the migration
+
+Please be aware that though GreptimeDB supports the wire protocol of MySQL, it does not mean GreptimeDB implements all
+MySQL's features. You may refer to the "[ANSI Compatibility](/reference/sql/compatibility.md)" to see the
+constraints regarding using SQL in GreptimeDB.
+
+## Migration steps
+
+### Create the databases and tables in GreptimeDB
+
+Before migrating the data from MySQL, you first need to create the corresponding databases and tables in GreptimeDB.
+GreptimeDB has its own SQL syntax for creating tables, so you cannot directly reuse the table creation SQLs that are produced
+by MySQL.
+
+When you write the table creation SQL for GreptimeDB, it's important to understand
+its "[data model](/user-guide/concepts/data-model.md)" first. Then, please take the following considerations in
+your create table SQL:
+
+1. Since the time index column cannot be changed after the table is created, you need to choose the time index column
+   carefully. The time index is best set to the natural timestamp when the data is generated, as it provides the most
+   intuitive way to query the data, and the best query performance. For example, in the IOT scenes, you can use the
+   collection time of sensor data as the time index; or the occurrence time of an event in the observability scenes.
+2. In this migration process, it's not recommend to create another synthetic timestamp, such as a new column created
+   with `DEFAULT current_timestamp()` as the time index column. It's not recommend to use the random timestamp as the
+   time index either.
+3. It's vital to set the most fit timestamp precision for your time index column, too. Like the chosen of time index
+   column, the precision of it cannot be changed as well. Find the most fit timestamp type for your
+   data set [here](/reference/sql/data-types#data-types-compatible-with-mysql-and-postgresql).
+4. Choose the most fit primary key columns based on your query patterns. Primary key columns store the metadata that is
+   commonly queried. The values in primary key columns are labels attached to the collected sources, generally used to
+   describe a particular characteristic of these sources. Primary key columns are indexed, making queries on them
+   performant.
+   > Note that in GreptimeDB, "primary key" and "tag" are used interchangeably. Please refer to
+   our "[data model](/user-guide/concepts/data-model.md)" for more details.
```

Comment:
```suggestion
4. Choose the most fit tag columns based on your query patterns. The tag columns store the metadata that is
   commonly queried. The values in the tag columns are labels attached to the collected sources, generally used to
   describe a particular characteristic of these sources. The tag columns are indexed, making queries on them
   performant.
```

* Please do not use the `primary key` word. In the data model of GreptimeDB, there is no column with type `Primary Key`, the column type should be `Tag`.
* We already told users need to understand the data model at line 20, and already described that the primary key specified the tag columns in both the data model document and the create statements document. So no need to note this in every documents, which increases the cost of  content maintenance.